dataland-batch-manager/src/main/kotlin/org/dataland/datalandbatchmanager/service/GleifApiAccessor.kt

```diff
@@ -35,6 +37,17 @@ class GleifApiAccessor(
         downloadFileFromGleif("latest.csv", targetFile, "full Golden Copy File")
     }
 
+    /**
+     * Downloads the latest complete Lei-ISIN mapping file
+     * @param targetFile the local target file to be written
+     */
+    fun getIsinMappingFile(targetFile: File) {
+        val apiText = URL(gleifMappingBaseUrl).readText()
+        val downloadLink = URL(apiText.split("\"downloadLink\":\"")[1].split("\"")[0])
```

Comment:
Is this approach to get the downloadLink safe? A proper JSON parsing would be. However, I think this is probably fine.